### PR TITLE
MTM-56141 Update MQTT Service documentation with all necessary information about private preview

### PR DESCRIPTION
**Pull request by slc-softwareag**

### CODE REVIEW

**Review by sagscmi — APPROVED** [review]
All good for me.  There will be more changes needed when the preview goes live, as well as Stefan's proposed re-work in MTM-56721, but none of that is blocking right now.

**Comment by BeateRixen** [comment]
@slc-softwareag This does not actually need approval as it is part of another PR.

File: content/device-integration/mqtt-service-bundle/overview.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ layout: redirect
 title: Overview
 ---
 
+{{< c8y-admon-preview >}}
+This feature is in Private Preview, that is, it is not enabled by default and maybe subject to change in the future.
+Please reach out to [product support](/additional-resources/contacting-support/) if interested.
```
Comment (BeateRixen) [inline content/device-integration/mqtt-service-bundle/overview.md:9]:
```suggestion
Please reach out to the [product support](/additional-resources/contacting-support/) if you are interested in this feature.
```

File: content/device-integration/mqtt-service-bundle/implementation.md [hunk 2]
```diff
@@ -81,13 +81,17 @@ both message header and body. The header size varies, but its minimum is 2 bytes
...
 
-#### Authentication {#authentication}
+#### Authentication and authorization {#authentication-and-authorization}
 
 Authentication types supported by MQTT Service are:
 
 *   Username and password. The MQTT username must include the tenant ID and username in the format `<tenantID>/<username>`.
 *   Device certificates - not yet supported. This will be added in a future release.
 
+{{< c8y-admon-important >}}
```
Comment (BeateRixen) [inline content/device-integration/mqtt-service-bundle/implementation.md:91]:
```suggestion
{{< c8y-admon-req >}}
```

File: content/device-integration/mqtt-service-bundle/implementation.md [hunk 3]
```diff
@@ -81,13 +81,17 @@ both message header and body. The header size varies, but its minimum is 2 bytes
...
+MQTT user which is used to connect to the MQTT Service must have the `MQTT_SERVICE_ADMIN` role assigned.
```
Comment (BeateRixen) [inline content/device-integration/mqtt-service-bundle/implementation.md:92]:
```suggestion
The MQTT user which is used to connect to the MQTT Service must have the `MQTT_SERVICE_ADMIN` role assigned.
```

File: content/device-integration/mqtt-service-bundle/implementation.md [hunk 4]
```diff
@@ -81,13 +81,17 @@ both message header and body. The header size varies, but its minimum is 2 bytes
...
+{{< /c8y-admon-important >}}
```
Comment (BeateRixen) [inline content/device-integration/mqtt-service-bundle/implementation.md:93]:
```suggestion
{{< /c8y-admon-req >}}
```

**Pull request merged**